test/functional/Java8andUp/playlist.xml

```diff
@@ -861,6 +861,7 @@
 	</test>
 
 	<test>
+		<aot>nonapplicable</aot>
```

Comment:
Please move `<aot>` tag after `<command>`, so that people will see test name and other more important tags first.